docs/chapters/patches.xml

```diff
@@ -52,6 +52,15 @@
     the your &debian-branch;, of course.
   </para>
 
+  <para>
+    Also, beware that &gbp-pq; currently has incomplete support for
+    <ulink url="http://dep.debian.net/deps/dep3/">DEP3</ulink> headers,
+    such as <computeroutput>Origin</computeroutput> and
+    <computeroutput>Forwarded</computeroutput>. These will be replaced
+    with <command>git-format-patch(1)</command> compatible options during
+    export.
+  </para>
```

Comment:
As far as I read the patch 'author' and 'description' are handled separately while others might be dropped. Isn't this the case? Can you add a testcase for Origin and Forwarded?